service/controller/v2/key/key.go

```diff
@@ -50,8 +50,11 @@ func CustomObjectHash(customObject v1alpha1.CertConfig) (string, error) {
 		return "", microerror.Mask(err)
 	}
 
-	h := sha1.New()
-	h.Write(b)
+	h := sha256.New()
```

Comment:
Note that the implication of this is that all certificates will be rolled. Shouldn't hurt, just saying.